docs/content/releases.md

```diff
@@ -6,6 +6,70 @@ doctypes: ["concept"]
+
+
+FEATURES:
+
+* [3143](https://github.com/nginxinc/kubernetes-ingress/pull/3143) Add annotations for deployment and daemonset.
+* [3260](https://github.com/nginxinc/kubernetes-ingress/pull/3260) Add support for EndpointSlices.
+* [3299](https://github.com/nginxinc/kubernetes-ingress/pull/3299) Support Dynamic namespaces using Labels.
+* [3261](https://github.com/nginxinc/kubernetes-ingress/pull/3261) Deep service insight endpoint for virtual server.
+* [3361](https://github.com/nginxinc/kubernetes-ingress/pull/3361) Add healthcheck for transport server.
+* [3347](https://github.com/nginxinc/kubernetes-ingress/pull/3347) Import JWKS from URL on JWT policy.
+* [3274](https://github.com/nginxinc/kubernetes-ingress/pull/3274) Allow configuration of map-hash-bucket-size and map-hash-max-size directives.
+* [3376](https://github.com/nginxinc/kubernetes-ingress/pull/3376) NSM OSS integration.
+
+
+IMPROVEMENTS:
+
+* [3170](https://github.com/nginxinc/kubernetes-ingress/pull/3170) Watch subset of namespaces for secrets. Thanks to [Hans Feldt](https://github.com/hafe).
+* [3341](https://github.com/nginxinc/kubernetes-ingress/pull/3341) Set value of `$remote_addr` to client IP when TLSPassthrough and Proxy Protocol are enabled.
+* [3131](https://github.com/nginxinc/kubernetes-ingress/pull/3131) NAP DoS images are now available in the AWS Marketplace.
+* [3231](https://github.com/nginxinc/kubernetes-ingress/pull/3231) Always print build info at the start.
+* [2735](https://github.com/nginxinc/kubernetes-ingress/pull/2735) Support default client proxy headers to be overwritten in VirtualServer. Thanks to [Alex Wied](https://github.com/centromere)
+
+
+FIXES:
+
+* [3139](https://github.com/nginxinc/kubernetes-ingress/pull/3139) Remove all IPV6 listeners in ingress resources with `-disable-ipv6` command line.
+* [3239](https://github.com/nginxinc/kubernetes-ingress/pull/3239) Fix git command to retrieve commit matching the tag.
+
+
+HELM CHART:
+
+* [2113](https://github.com/nginxinc/kubernetes-ingress/pull/3113) Add JSON Schema.
+* [3143](https://github.com/nginxinc/kubernetes-ingress/pull/3143) Add annotations for deployment and daemonset.
+* [3136](https://github.com/nginxinc/kubernetes-ingress/pull/3136) Add controller.dnsPolicy. Thanks to [Dong Wang](https://github.com/wd).
+* [3065](https://github.com/nginxinc/kubernetes-ingress/pull/3065) Add annotations to the service account. Thanks to [0m1xa](https://github.com/0m1xa).
+* [3276](https://github.com/nginxinc/kubernetes-ingress/pull/3276) Add horizontalpodautoscaler. Thanks to [Bryan Hendryx](https://github.com/coolbry95).
+
+
+UPGRADE:
+* For NGINX, use the 3.0.0 images from our [DockerHub](https://hub.docker.com/r/nginx/nginx-ingress/tags?page=1&ordering=last_updated&name=3.0.0), [GitHub Container](https://github.com/nginxinc/kubernetes-ingress/pkgs/container/kubernetes-ingress) or [Amazon ECR Public Gallery](https://gallery.ecr.aws/nginx/nginx-ingress).
```

Comment:
I think we should mention here that users needs to make sure their k8s version is 1.21+ before upgrading to 3.0.0